app/src/main/java/org/projectbuendia/client/providers/Contracts.java

```diff
@@ -36,7 +36,8 @@
         ORDERS("orders"),
         PATIENTS("patients"),
         USERS("users"),
-        SYNC_TOKENS("sync_tokens");
+        SYNC_TOKENS("sync_tokens"),
+        UNSYNC_TOKENS("unsync_forms");
```

Comment:
Agree on (1). @viniciusboson can you add a comment noting that Order Execution is synchronous-only and the reason why (to prevent double administering of treatments)

On (2) I also agree it would be especially confusing if (A) and (B) were both on the same patient. If we're going to make it attempt submit queued submissions when submitting a form, we might as well send them all.

@zestyping: there is technical design documentation in this document: https://docs.google.com/document/d/1ah6Ezxc7Fhh3CcmoWTPUhbkYCCkiOnPbGHJ5038u7dc/edit